distribution/scripts/jmeter/performance-test-scenarios.sh

```diff
@@ -48,6 +49,7 @@ declare -A test_scenario0=(
 declare -A test_scenario1=(
     [name]="CBRProxy"
     [display_name]="CBRProxy"
+    [description]="Routing message by reading a text field in message body"
```

Comment:
Shall we change to "Routing the message based on the content of the message body"

Reply:
Changed